### PR TITLE
Add Warzone

### PR DESCRIPTION
**Pull request by TheRealPear**
## Type of change

- [X] Add new directory for a server
- [ ] Update directory of a server
- [ ] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [X] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [X] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [X] I have created the manifest.json with the required values

## Further comments
Server is currently online and is behind a proxy, you may need to refresh a few times before it goes through

### CODE REVIEW

File: minecraft_servers/warzone/manifest.json [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    "direct_ip": "play.warzone.network",
+    "server_wildcards": [
+        "%.warzone.network",
+        "%.warz.one"
+    ],
+    "supported_languages": [
+        "en"
+    ],
+    "social": {
+        "web": "https://warzone.network",
```
Comment (DoJapHD) [inline minecraft_servers/warzone/manifest.json:13]: Many things just load indefinitely

File: minecraft_servers/warzone/logo.png
Comment (DoJapHD) [inline minecraft_servers/warzone/logo.png:1]: Invalid logo size! Size is 255x51px, shortest side must be at least 128px

File: minecraft_servers/warzone/manifest.json [hunk 2]
```diff
@@ -0,0 +1,60 @@
+{
+    "server_name": "warzone",
+    "nice_name": "Warzone Network",
+    "direct_ip": "play.warzone.network",
+    "server_wildcards": [
+        "%.warzone.network",
```
[the remark this answers is not in the document]
Comment (TheRealPear) [reply minecraft_servers/warzone/manifest.json:6]: Hi, it's a bad timing on my part. The domain apparently stopped working around the same time I submitted this and the infrastructure hosting the server is down for maintenance. Would it be fine if you could keep this pull request on hold or should I close it for now?

File: minecraft_servers/warzone/manifest.json (hunk 1 above)
Comment (TheRealPear) [reply minecraft_servers/warzone/manifest.json:13]: This is a known issue, the website is mostly used as a hub to our store, Discord invitation, and rules.

File: minecraft_servers/warzone/manifest.json [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    "discord": {
+        "server_id": 333669866735271940,
+        "rename_to_minecraft_name": false
+    },
+    "brand": {
+        "primary": "#2F3136",
+        "background": "#9148D3",
+        "text": "#FFFFFF"
+    },
+    "user_stats": "https://warzone.network/p/@{userName}"
```
[the remark this answers is not in the document]
Comment (TheRealPear) [reply minecraft_servers/warzone/manifest.json:59]: I will remove this for now.

**Comment by DoJapHD** [comment]
Inactive PR

**Pull request closed without merging**